python/cuopt/cuopt/linear_programming/problem.py

```diff
@@ -0,0 +1,796 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.  # noqa
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enum import Enum
+
+import numpy as np
+
+import cuopt.linear_programming.data_model as data_model
+import cuopt.linear_programming.solver as solver
+import cuopt.linear_programming.solver_settings as solver_settings
+
+
+# The type of a variable is either continuous or integer
+class VType(str, Enum):
+    CONTINUOUS = "C"
+    INTEGER = "I"
+
+
+# The sense of a constraint is either LE, GE or EQ
+class CType(str, Enum):
+    LE = "L"
+    GE = "G"
+    EQ = "E"
+
+
+# The sense of a model is either MINIMIZE or MAXIMIZE
+class sense(int, Enum):
+    MAXIMIZE = -1,
+    MINIMIZE = 1
+
+
+class Variable:
+    """
```

Comment:
Lets add more details to the doc, similar to this https://github.com/NVIDIA/cuopt/blob/455bdb0b4f99b64e978f01b44fe37d7dd558308a/python/cuopt_self_hosted/cuopt_sh_client/cuopt_self_host_client.py#L246

Reply:
The Variable is not supposed to be initialized directly. The main way to do is to use addVariable() function in the Problem Class, the parameters have been defined there.